deploy/docs/standAlonePrometheus.md

```diff
@@ -1,6 +1,12 @@
 # How to install if you have standalone Prometheus
 
-Update your Prometheus configuration file’s `remote_write` section, as per the documentation [here](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write), by taking the `remoteWrite` section of the `prometheus-overrides.yaml` file, and making the following changes:
+1. Download the Prometheus Operator `prometheus-overrides.yaml` by running
+
+```bash
+$ curl -LJO https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/v0.17.0/deploy/helm/prometheus-overrides.yaml
```

Comment:
Actually this line should be:
```
http$ curl -LJO  s://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/release-v0.17/deploy/helm/prometheus-overrides.yaml
```
(`release-v0.17` instead of `v0.17.0`)

Reply:
We need to fix this in both master and in here and in all of the files, but that can be another PR.